### DISCUSSION TITLE
schemaDoc for custom GraphQL schema

### DISCUSSION THREAD

**Discussion by AntonAL** [keystonejs/keystone#7258] [Questions] [answered]
Hello!

I need to write textual documentation for custom queries/mutations, but did not found a way to do in in config, passed to [keystone.extendGraphQLSchema](https://v5.keystonejs.com/keystonejs/keystone/#extendgraphqlschemaconfig).

I’m expecting an option like `schemaDoc`, that is available in options, passed to `keystone.createList`.

Finally, i need it to be reflected in GraphiQL UI for custom query or mutation.
Anyone knows, how to do it?

**Comment by raveling** [comment, marked as answer]
Where you write descriptions for queries and mutations in schema extensions depends on how you're extending your schema. If you're using the `graphQLSchemaExtension` function that accepts SDL strings - see this example:

https://github.com/keystonejs/keystone/blob/f0e9193c5f1594cda61397318f1ce6c4afd54160/examples/extend-graphql-schema/custom-schema.ts#L7
